### PR TITLE
Drivetrain and Joystick

### PR DESCRIPTION
**Pull request by LinusSkucas**
This Pull Request closes issue #

Description of the changes:
 - 

 - [x] I have labelled this issue with the priority and correct subsystem.
 - [x] I set the person that I worked on this with as a reviewer (who will need to approve the changes)

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/MecanumDrive.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  private boolean inverted;
+  private CANSparkMax[] motors;
+
+  private final MecanumDriveKinematics kinematics;
+
+  public MecanumDrive() {
+    this.motors = new CANSparkMax[4];
+    this.motors[0] = new CANSparkMax(Constants.WHEEL_PORT_FRONT_LEFT, MotorType.kBrushless);
+    this.motors[1] = new CANSparkMax(Constants.WHEEL_PORT_FRONT_RIGHT, MotorType.kBrushless);
+    this.motors[2] = new CANSparkMax(Constants.WHEEL_PORT_REAR_LEFT, MotorType.kBrushless);
```
Comment (LinusSkucas) [inline src/main/java/frc/robot/subsystems/MecanumDrive.java:24]: Don't use arrays

Comment (SBsteveyHops) [reply src/main/java/frc/robot/subsystems/MecanumDrive.java]: Fixed

**Review by FlareTheSaberDragon — APPROVED** [review]
that looks good

**Review by SBsteveyHops — COMMENTED** [review]
I think it's ready

**Comment by LinusSkucas** [comment]
@SBsteveyHops you'll need to approve it for it to go through

**Comment by SBsteveyHops** [comment]
it's not letting me

**Review by SBsteveyHops — APPROVED** [review]

**Pull request merged**